docs/contributing.md

```diff
@@ -73,6 +73,13 @@ pytest tests/ --no-cov
     If you are debugging failing tests using the `--pdb` flag, tests will only run on one thread instead of the default (which is the maximum number of threads your machine has available).
     This will slow down your tests, so do not use `--pdb` unless you are actively debugging.
 
+## Updating the project when the template updates
+
+This project has been built with [cruft](https://cruft.github.io/cruft/) based on the [Arup Cookiecutter template](https://github.com/arup-group/cookiecutter-pypackage).
+When changes are made to the base template, they can be merged into this project by running `cruft update` from the  `genet` mamba environment.
+
+You may be prompted to do this when you open a Pull Request, if our automated checks identify that the template is newer than that used in the project.
```

Comment:
Easiest to have this check in the CI as there isn't a bot that will do it automatically. CI actions outside PRs are easily missed so I put it in with the PR checks. Reviewers can choose to ignore a failing CI check if they want to tackle updating against the template in a separate PR (which, I agree, is a cleaner way to do it)

Reply:
Looks like there might be a bot on its way: https://github.com/renovatebot/renovate/discussions/24000

Reply:
Had a look at doing this with our own action. It _would_ be possible, but the action would need quite a lot of repo write permissions that I wouldn't be comfortable with. Also, since it might need to update CI workflows as part of the process, extra steps are needed per repo to create a token that would allow that (and having to go through that process is part of the reason I'm uncomfortable with it...).